src/platform/user/profile/vet360/tests/containers/AddressValidationView.unit.spec.jsx

```diff
@@ -0,0 +1,417 @@
+import React from 'react';
```

Comment:
mostly copied from the existing tests for the `AddressValidationModal` component.